pkg/scheduler/algorithm/predicates/testing_helper.go

```diff
@@ -19,8 +19,9 @@ package predicates
 import (
 	"fmt"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	storagev1 "k8s.io/api/storage/v1"
+	storagev1beta1 "k8s.io/api/storage/v1beta1"
```

Comment:
Could we remove storagev1beta1 and use storagev1 here?

Reply:
Not yet, `CSINode` object is beta at the moment